### PR TITLE
Add request telemetry

### PR DESCRIPTION
**Pull request by nick-saxelby-cko**
- Adds request latency telemetry by default
- Allows user to opt-out

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -240,6 +240,29 @@ The execution of integration tests require the following environment variables s
 * For Default account systems (OAuth): `CHECKOUT_DEFAULT_OAUTH_CLIENT_ID` & `CHECKOUT_DEFAULT_OAUTH_CLIENT_SECRET`
 * For Previous account systems (ABC): `CHECKOUT_PREVIOUS_PUBLIC_KEY` & `CHECKOUT_PREVIOUS_SECRET_KEY`
 
+## Telemetry
+Request telementry is enabled by default in the .NET SDK. Request latency is included in the telemetry data. Recording the request latency allows Checkout.com to continuously monitor and imporove the merchant experience.
```
Comment (taylor-smith-cko) [inline README.md:244]: Request telemetry*

File: src/CheckoutSdk/ApiClient.cs [hunk 2]
```diff
@@ -19,16 +21,21 @@
         private readonly ILogger _log = LogProvider.GetLogger(typeof(ApiClient));
 #endif
 
+        private static string sdkTelemetryHeader = "cko-sdk-telemetry";
```
[the remark this answers is not in the document]
Comment (nick-saxelby-cko) [reply src/CheckoutSdk/ApiClient.cs]: Fixed in 9873390c98299f0057a68ff72f25b09118d6c004

File: src/CheckoutSdk/ApiClient.cs [hunk 3]
```diff
@@ -19,16 +21,21 @@
...
+        private static int maxCountInTelemetryQueue = 10;
```
[the remark this answers is not in the document]
Comment (nick-saxelby-cko) [reply src/CheckoutSdk/ApiClient.cs]: Fixed in 9873390c98299f0057a68ff72f25b09118d6c004

File: src/CheckoutSdk/ApiClient.cs [hunk 4]
```diff
@@ -19,16 +21,21 @@
...
         private readonly HttpClient _httpClient;
         private readonly Uri _baseUri;
         private readonly ISerializer _serializer = new JsonSerializer();
+        private ConcurrentQueue<RequestMetrics> requestMetricsQueue = new ConcurrentQueue<RequestMetrics>();
```
[the remark this answers is not in the document]
Comment (nick-saxelby-cko) [reply src/CheckoutSdk/ApiClient.cs]: Fixed in 9873390c98299f0057a68ff72f25b09118d6c004

**Pull request merged**